### ISSUE TITLE
[moreuserinfo] - Dont show details mail config, dont show calendars info

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#503]
```
Token: dd12c1d22a08aff4be5253f5be23c247 (Don't modify this token!)
Version: 4.0.6 (28-03-2013)
PHP: 5.3.3
RCMAIL: 0.8.6
Database: mysql
SERVER: Apache/2.2.3 (CentOS)
----
I.  Issue Description:
Plugin dont show mail details and calendars
II. Steps to reproduce the Issue:
1. Add in RC config(plugins) 'moreuserinfo'
2. Change moreuserinfo config:
$rcmail_config['moreuserinfo'] = array(
  'IMAP' => array( // Service
    'default' => array( //Domain (use 'default' if your user don't have a domain part)
      'encryption' => 'SSL',
      'server' => 'imap.mydomain.tld',
      'port' => '993',
      'auth' => 'yes'
    ),
  ),
  'SMTP' => array(
    'default' => array(
      'encryption' => 'no',
      'server' => 'smtp.mydomain.tld',
      'port' => '25',
      'auth' => 'yes'
    ), // NOTE: In this config example mydomain2.tld does not offer SMTP service
  ),
);
3.Dont show mail details or calendar info
```

Original issue reported on code.google.com by `mmartine...@gmail.com` on 3 Apr 2013 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
I delete user name and identity
```

Original comment by `mmartine...@gmail.com` on 3 Apr 2013 at 9:11

Attachments:
- [issue.png](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-503/comment-1/issue.png)

**Comment by GoogleCodeExporter** [comment]
```
Obiously a mis-configuration. Your users have a domain part.
```

Original comment by `roland.l...@gmail.com` on 4 Apr 2013 at 6:24
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
I have this errors in my log:

[04-Apr-2013 14:29:51] MDB2 Error: no such table (-18): _doQuery: [Error 
message: Could not execute statement]
[Last executed query: PREPARE 
mdb2_statement_mysql_508d3e7d6e557f268a60106e94ecaad9106b0c282 FROM 'SELECT 
user_id, password, url, label from carddav_server WHERE user_id=?']
[Native code: 1146]
[Native message: Table 'roundcubemailtest.carddav_server' doesn't exist]

[04-Apr-2013 14:29:51 -0500]: DB Error: _doQuery: [Error message: Could not 
execute statement]
[Last executed query: PREPARE 
mdb2_statement_mysql_508d3e7d6e557f268a60106e94ecaad9106b0c282 FROM 'SELECT 
user_id, password, url, label from carddav_server WHERE user_id=?']
[Native code: 1146]
[Native message: Table 'roundcubemailtest.carddav_server' doesn't exist]
 in /usr/share/roundcubemail086Test/program/include/rcube_mdb2.php on line 296 (GET /?_task=settings&_action=plugin.moreuserinfo&_framed=1)

```

Original comment by `mmartine...@gmail.com` on 4 Apr 2013 at 7:35

**Comment by GoogleCodeExporter** [comment]
```
I dont have installed cardDav plugin. I dont know why the plugin ask for 
carddav_server .
```

Original comment by `mmartine...@gmail.com` on 4 Apr 2013 at 7:37

**Comment by GoogleCodeExporter** [comment]
```
I discovered the problem is related with stored preferences in users tables. I 
delete preference and this solve the problem.
```

Original comment by `mmartine...@gmail.com` on 4 Apr 2013 at 10:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `roland.l...@gmail.com` on 8 Apr 2013 at 5:46
- Changed state: **Re-Opened**

**Comment by GoogleCodeExporter** [comment]
```
Fixed Comment #3 issue.
```

Original comment by `roland.l...@gmail.com` on 8 Apr 2013 at 5:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `roland.l...@gmail.com` on 8 Apr 2013 at 5:48
- Changed state: **Fixed**
